DeepLMTProvider/Sdl.Community.DeelLMTProvider/Studio/DeepLMtTranslationProviderFactory.cs

```diff
@@ -1,9 +1,8 @@
 ﻿using System;
-using Sdl.Community.DeepLMTProvider.WPF;
-using Sdl.Community.DeepLMTProvider.WPF.Model;
+using Sdl.Community.DeepLMTProvider.Model;
 using Sdl.LanguagePlatform.TranslationMemoryApi;
 
-namespace Sdl.Community.DeepLMTProvider
+namespace Sdl.Community.DeepLMTProvider.Studio
```

Comment:
Because of a problem in Plugin Fremework if we change the namespace when the user will download the latest version via AppStore integration he'll end up in the unpacked folder with dlls from the old version and for the new one with the name changed. Please leave the namespace of the plugin unchanged